README.md

```diff
@@ -164,14 +166,22 @@ export default App;
 
     - Android:
 
-      	```bash
+          ```bash
+
       $ react-native run-android
+
+      ```
```

Comment:
looks like these should be removed as the end of the code formatting is already defined l. 175